packages/test/test-end-to-end-tests/src/test/gc/gcTombstoneDataStores.spec.ts

```diff
@@ -602,6 +625,52 @@ describeNoCompat("GC data store tombstone tests", (getTestObjectProvider) => {
 			},
 		);
 
+		// If this test starts failing due to runtime is closed errors try first adjusting `sweepTimeoutMs` above
+		itExpects(
+			"Requesting tombstoned datastores succeeds with future gcEnforcementCurrentValue",
+			[
+				// Interactive client's request that succeeds
+				{
+					eventName: "fluid:telemetry:ContainerRuntime:GC_Tombstone_DataStore_Requested",
+					category: "generic",
+					gcEnforcementDisabled: true,
+				},
+				{
+					eventName:
+						"fluid:telemetry:ContainerRuntime:GarbageCollector:SweepReadyObject_Loaded",
+				},
+			],
+			async function () {
+				// Note: The Summarizers in this test don't use the "future" GC option - it only matters for the interactive client
```

Comment:
Hmmm... I'd like to test that I can persist the value and then use a different one later, rather than depending on undefined.  Which makes me realize - I'm not persisting the given value!  Will fix tomorrow.

Also, are Summarizer clients given the same RuntimeOptions as the main clients?  I assume so - I mean, it must be, we've been using options for GC for a while.